src/constants/index.ts

```diff
@@ -856,3 +856,13 @@ export const workType = [
   'Fulltime',
   'Internship',
 ];
+
+export const bountyDescriptionFallback = [
```

Comment:
rename this to `listingDescriptionHeadings`, and move it this to `CreateListing.tsx` itself